monster.hpp

```diff
@@ -1,28 +1,32 @@
 #ifndef MONSTER_H
 #define MONSTER_H
+
 #include <string>
+#include <fstream>
 
 class Monster {
 
-  const std::string name;
+private:
+
+  std::string name;
   int hp;
-  const int dmg;
+  int dmg;
```

Comment:
Miért tűntek el a `const`ok? Továbbra sem módosíthatók ezek az adattagok a létrehozás után. A parsolásnál a megfelelő értékekkel kell létrehozni a példányokat, nem utólag módosítani azok értékeit.